modules/acquia_telemetry/src/Event.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Drupal\acquia_telemetry;
+
+use JsonSerializable;
```

Comment:
`JsonSerializable` is in the global namespace, so there is no need for this `use` statement. The `implements` clause will need to be changed to `\JsonSerializable`, though.